cmd/osv-scanner/main_test.go

```diff
@@ -508,8 +508,19 @@ func TestRun_LocalDatabases(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
 
-			testDir, cleanupTestDir := createTestDir(t)
-			defer cleanupTestDir()
+			var testDir string
+			var cleanupTestDir func()
+
+			if os.Getenv("TEST_ACCEPTANCE") != "true" {
```

Comment:
we should use the `testutility.AcceptanceTests` for this

Reply:
can do, is there an example of the way it should be used? I didn't think it could be used like that cos I couldn't find any example in the code, might just be looking for the wrong thing though.

Reply:
https://github.com/ackama/osv-scanner/blob/ff8aee76ad1ea6d56a2e7e8aac88d3deeee58566/internal/sourceanalysis/rust_test.go#L68-L68

Reply:
oh I thought this would just skip the test completely does it not?

Reply:
I think its just skips the tests from messing with it, but I could build something in the utility so that it is only accessing TEST_ACCEPTANCE in one place?

Reply:
ah yup you're totally right - let's introduce a new `IsRunningAcceptanceTests` helper which we can use in the existing helper plus here